src/pages/Home/VideoCards/VideoCards.jsx

```diff
@@ -15,12 +15,12 @@ const VideoCard = styled.div`
   }
 `;
 
-const VideoCardInfo = styled.div`
+export const VideoCardInfo = styled.div`
   padding: 20px;
   width: 310px;
 `;
 
-const youtubeCards = mockYoutubeData.items.map((elem) => {
+export const youtubeCards = mockYoutubeData.items.map((elem) => {
```

Comment:
Try the following instead, so you could reuse them
```suggestion
export const YoutubeCards = (youtubeItems) => (youtubeItems.map((elem) => <VideoCard key={elem.etag}>))
) 
```

in which the props for this case, are mockYoutubeData.items